datadog_checks_base/tests/test_http.py

```diff
@@ -414,6 +415,52 @@ def test_unknown_name_default(self):
         assert http.options['verify'] is True
 
 
+class TestLogger:
+    def test_default(self, caplog):
+        check = AgentCheck('test', {}, [{}])
+
+        with mock.patch('requests.get'):
```

Comment:
Not sure what caplog default level is, but don't we need to first set caplog level to DEBUG? Otherwise the test might pass even though the request is being logged.